source/MaterialXGenGlsl/GlslShaderGenerator.cpp

```diff
@@ -283,23 +283,23 @@ ShaderPtr GlslShaderGenerator::generate(const string& shaderName, ElementPtr ele
     if (!vsConstants.empty())
     {
         shader.addComment("Constant block: " + vsConstants.name);
-        emitVariableBlock(vsConstants, _syntax->getConstantQualifier(), shader);
+        emitVariableBlock(vsConstants, _syntax->getConstantQualifier(), ShaderGenerator::SEMICOLON_NEWLINE, shader);
```

Comment:
The ShaderGenerator:: prefix is not needed as this is a sub-class to ShaderGenerator.